.github/workflows/update-github.yaml

```diff
@@ -0,0 +1,45 @@
+name: "UpdateOctokit"
+
+on:
+  schedule:
+    - cron: '0 18 * * 0'
+
+jobs:
+  UpdateOctokit:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Update Octokit
+      working-directory: packages/github 
+      run: |
+        npx npm-check-updates -u --dep prod
+        npm install
+    - name: Check Status
+      id: status
+      working-directory: packages/github 
+      run: | 
+        if [[ "$(git status --porcelain)" != "" ]]; then 
+        echo "::set-output name=createPR::true"
+        git config --global user.email "bot@github.com"
+        git config --global user.name "Dependency Update Bot"
+        git checkout -b bots/updateGitHubDependencies-${{github.run_number}}
+        git add .
+        git commit -m "Update Dependencies"
+        git push --set-upstream origin bots/updateGitHubDependencies-${{github.run_number}}; fi
+    - name: Create PR
+      if: ${{steps.status.outputs.createPR}}
+      uses: actions/github-script@v2
+      with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
fyi, it looks like you could omit `github-token: ...` since `${{ github.token }}` is [the default](https://github.com/actions/github-script/blob/v2/action.yml#L13)